src/main/java/com/sap/sgs/phosphor/fosstars/model/Value.java

```diff
@@ -51,13 +51,21 @@
   T get();
 
   /**
-   * Call a processor to process the value if the value is known.
+   * Call a processor to process the value if it's known.
    *
    * @param processor The processor to be called.
    * @return This Value instance.
    */
   Value<T> processIfKnown(Processor<T> processor);
```

Comment:
Not directly related to this commit but the `Processor` interface could have the  `@FunctionalInterface` annotation.

Reply:
Okay, let's add it.